src/shortcuts.js

```diff
@@ -0,0 +1,18 @@
+const createShortcut = method => ({ options = {}, ...rest} = {}) => ({
+    ...rest,
+    options: {
+        ...options,
+        method
```

Comment:
now that i think about, it was a wrong decision, i can see how overriding the method when calling api may be beneficial
i'll change that